### DISCUSSION TITLE
Downgrading Project

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#8341] [Archive]
Good morning guys,

I made a mistake and accidentally updated a project in production to version rc.94, but this version has a problem in the data field. does not let you select month, day, year, and this field is used a lot in the project. tried downgrading to rc.93 version but it doesn't work, any solution?

To downgrade I followed the steps in the manual:
npm install directus@v9.0.0-rc.93 
directus database migrate:down

![image](https://user-images.githubusercontent.com/81326296/134903258-433dea66-234b-4a82-ac4a-1bcd122006d5.png)

**Comment by joselcvarela** [comment]
Hey @hugopvl 
Didn't you backup your data before the upgrade? 😨
I hope so, as you can expect if you need to solve this issue you will may need to install directus locally.

**Reply by hugopvl** [reply]
@joselcvarela 
Yes i did the backup of the project folder, but not the database :/ and i did the database migration :/

Now if you run the old backup project it gives the same error.

Any idea how to resolve this situation?

**Reply by joselcvarela** [reply]
@hugopvl can you share the error on the request?
We need that to find the source of the problem.

**Reply by joselcvarela** [reply]
@hugopvl meanwhile you can use "raw value" to fulfil the dates.

**Comment by rijkvanzanten** [comment]
Heya!

`migrate:down` currently undoes the last "created" migration, instead of the migration that was last applied (this is tracked in https://github.com/directus/directus/issues/8254 and will be fixed shortly). 

As a solution in the meantime, you can either try to manually undo the last couple migrations by [checking what it does](https://github.com/directus/directus/blob/main/api/src/database/migrations/20210910A-move-module-setup.ts) and removing the row from directus_migrations, _or_ and I would strongly recommend this: wait for the next release to come out later today/tomorrow, and use the fields _Raw Value_ option to modify the value directly:

<img width="368" alt="CleanShot 2021-09-27 at 09 48 22@2x" src="https://user-images.githubusercontent.com/9141017/134921443-b0f30d1d-6360-42a2-87eb-6225bb28e206.png">

**Reply by hugopvl** [reply]
@joselcvarela 
@rijkvanzanten 
Thank you very much for your attention, at the moment i think i'll solve it with raw value and i'm waiting for the new version.
Thank you so much guys.

**Reply by joselcvarela** [reply]
Hey @hugopvl, we have updated to rc95 about 6 hours ago.
You can upgrade your project ;)

**Reply by hugopvl** [reply]
Good morning guys,

when trying to upgrade to rc.95 version the following happened:

and I can't log in:

![image](https://user-images.githubusercontent.com/81326296/135053081-ba577412-9bde-498f-b42f-94f7ec532cca.png)
![image](https://user-images.githubusercontent.com/81326296/135053185-8de8efa0-17b8-4303-afaa-86232dc6ee09.png)

**Reply by azrikahar** [reply]
@hugopvl are you by any chance using mssql? With that said, please reply over at #8368 as it seems to be a similar error. That'll help us consolidate the error reports in one place 👍

**Reply by hugopvl** [reply]
Yes I am using MSsql Server
